src/components/Home.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import  { getEvents } from '../redux/events'
+import { getAllUsers } from '../redux/users'
+
+
+export class Home extends Component {
+  constructor() {
+    super()
+    this.state = {
+      events: [],
+      allUsers: []
+    }
+    this.createEventButton = this.createEventButton.bind(this)
+  }
+
+  async componentDidMount() {
+    await this.props.fetchAllUsers()
+    await this.props.fetchEvents()
+    this.setState({
+      events: this.props.events,
+      allUsers: this.props.allUsers
+    });
+  }
+  createEventButton () {
+    window.location.hash = "#/createEvent"
+  }
+  render() {
+
+    //Still need to connect users on redux
+    return (
+      <div className="main-container">
+        <div className="left-container">
+          <img src={`https://randomuser.me/api/portraits/women/${Math.floor(Math.random() * (40 - 1) + 1)}.jpg`}/>
```

Comment:
this file is getting a bit too big... consider making separate components for each "big" thing